src/types/line.js

```diff
@@ -65,6 +67,11 @@ export default class LineAnnotation extends Element {
   }
 
   // TODO: make private in v2
+  /**
+   * @param {boolean} useFinalPosition - use the element's animation target instead of current position
+   * @param {top, right, bottom, left} [chartArea] - optional, area of the chart
+   * @returns {boolean} true if the label is visible
+   */
```

Comment:
can you move the todo inside jsdoc in this one, because it now has jsdoc and it looks stupid :)

Reply:
:+1: